tutorials/quickstart/tutorial.md

```diff
@@ -8,9 +8,33 @@ extensions:
   nextText: "Next: Recovering from failure"
   nextUrl: tutorials/quickstart/tutorial-reliability
 previewImage: add-shipping-endpoint.png
+abTestName: Quickstart intro 2022 01
 ---
 
-include: quickstart-tutorial-intro-paragraph
```

Comment:
@sergioc @soujay @DavidBoike should the now redundant include file be removed as part of this PR?

Reply:
When the test ends it will be needed again, so I'm not sure.

Reply:
The include is also used here: https://github.com/Particular/docs.particular.net/blob/master/tutorials/index.md#nservicebus-quick-start